waterlog/src/Stylesheets/_progressBar.scss

```diff
@@ -33,13 +33,13 @@
 }
 
 .low {
-  
+  color: #FFEA00
```

Comment:
You used rgb earlier, please stick to rgb or hexadecimal

Reply:
Done , please re-check